### PR TITLE
Koala client-side destination

### PR DESCRIPTION
**Pull request by mshwery**
Hello!

This PR introduces a new client-side action destination for 🐨 

The implementation here is pretty basic: 
- we initialize the destination by including our tiny buffer script and load the umd bundle from a cdn
- we call `KoalaSDK.load` to create an instance of the SDK client (which gets passed into the actions)
- we have a couple actions that customers can configure
- we wire up a couple presets

## Testing

Tested locally via automated unit tests and by testing with `yarn build && cd /packages/browser-destinations && yarn dev`.

|Plugin Load|Breakpoints|Network!|
|:--:|:--:|:--:|
|Screen Shot 2022-03-14 at 5 42 18 PM|Screen Shot 2022-03-14 at 5 44 12 PM|Screen Shot 2022-03-14 at 5 44 24 PM|

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Review by pooyaj — APPROVED** [review]
🚢 🐨

File: packages/browser-destinations/src/destinations/koala/identifyVisitor/index.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      required: true,
+      default: { '@path': '$.traits' },
+      defaultObjectUI: 'object'
+    }
+  },
+  perform: (koala, { payload }) => {
+    if (payload?.traits) {
+      koala.identify(payload.traits).catch(() => {
+        // Ignore
+      })
```
[the remark this answers is not in the document]
Comment (pooyaj) [reply packages/browser-destinations/src/destinations/koala/identifyVisitor/index.ts:25]: Maybe slightly cleaner, but doesn't have much benefit from functionality standpoint. Possibly the only difference with returning the promise would be more accurate `plugin_time` in AJS. Can address in a follow-up PR if you like.

File: packages/browser-destinations/src/destinations/index.ts [hunk 2]
```diff
@@ -32,5 +32,6 @@ register('60fb01aec459242d3b6f20c1', './braze')
 register('60f9d0d048950c356be2e4da', './braze-cloud-plugins')
 register('6170a348128093cd0245e0ea', './friendbuy')
 register('6141153ee7500f15d3838703', './fullstory')
+register('<koala-destination-id>', './koala')
```
[the remark this answers is not in the document]
Comment (pooyaj) [reply packages/browser-destinations/src/destinations/index.ts:35]: Yup, please comment this one out. Will add when we get the `id`

**Review by dlasky — APPROVED** [review]

File: packages/browser-destinations/src/destinations/koala/identifyVisitor/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (mshwery) [reply packages/browser-destinations/src/destinations/koala/identifyVisitor/index.ts:25]: We could eliminate the `void` vs `catch` happening here and just let the framework handle it, too.

[the remark this answers is not in the document]
Comment (pooyaj) [reply packages/browser-destinations/src/destinations/koala/identifyVisitor/index.ts:25]: @mshwery perfect, let's make this change and then merge deploy this.

**Pull request merged**